dao/pom.xml

```diff
@@ -64,53 +64,19 @@
             <artifactId>httpclient</artifactId>
             <version>4.5.8</version>
         </dependency>
-        <dependency>
```

Comment:
Has this been ran on a proper hadoop environment and tested with oozie?

Reply:
The very same plugins were included in the master `'pom.xml`.  _Idea_ was pointing it out.
Compared and kept the main one. Bot no extra tests other then the ones included in the build has been done so far. What is the worry?

Reply:
Worried about all sorts of binary incompatibilities that these were trying to solve.

Though it's possible that these were no longer needed.